autorest/models/schema.py

```diff
@@ -24,14 +24,93 @@
 #
 # --------------------------------------------------------------------------
 
-class CompositeType:
-    """Represent a composite type schema ready to be serialized in Python.
+
+class ComposedProperty:
+    """Represents a property of a CompositeType
     """
+    def __init__(self, name, description, property_type, **kwargs):
```

Comment:
Feel free to use type annotation, this is Python 3.6 minimal

Reply:
ok, going to switch over the commenting